Makefile

```diff
@@ -1,18 +1,35 @@
+export COMPOSE_ENGINE=docker-compose
```

Comment:
Can we call this something like `CACHITO_COMPOSE_ENGINE` so I can set it in my `.bashrc` and forget about it? Then in this file we can set a default value if needed.

Reply:
Done